docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -24,7 +24,7 @@ Using the cluster wizard:
 On the next page:
 
 * (SaaS and remote self-hosted cluster only) Install a trusted certificate to the domain entered above.
-*  Run the [Helm](https://helm.sh/docs/intro/install/) command provided in the wizard.
+* Run the [Helm](https://helm.sh/docs/intro/install/) command provided in the wizard.
 
 ## Cluster Table
```

Comment:
@jasonnovichRunAI I think the new sections should be after "Verify your installation"